### PR TITLE
Bump version 2.9.6

### PR DESCRIPTION
**Pull request by tylerasai**
Bump to v2.9.6

Changelog update is pending

### CODE REVIEW

File: azure-container-apps-arm/README.md [hunk 1]
```diff
@@ -65,13 +65,14 @@ To finish setting up automated user provisioning, [connect your identity provide
 1. Within your deployed 1Password SCIM Bridge Container App in the [Azure Container Apps Portal](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.App%2FcontainerApps), select **Containers** from the sidebar.
 2. Click **Edit and deploy**.
 3. Select the checkbox next to your **op-scim-bridge** container, then choose **Edit**.
-4. Change the version number **2.9.5** in the **Image and Tag** field, **1password/scim:v2.9.5** to match the latest version from our [SCIM bridge releases notes website](https://releases.1password.com/provisioning/scim-bridge/).
+4. Change the version number **2.9.5** in the **Image and Tag** field, **1password/scim:v2.9.6** to match the latest version from our [SCIM bridge releases notes website](https://releases.1password.com/provisioning/scim-bridge/).
```
Comment (plttn) [inline azure-container-apps-arm/README.md:68]: Should the first reference here be 2.9.6 as well to keep with the pattern?

**Review by plttn — CHANGES_REQUESTED** [review]
Aside from whitespace changes, only one note

File: azure-container-apps-arm/README.md (hunk 1 above)
Comment (tylerasai) [reply azure-container-apps-arm/README.md]: Good point! Let's change it!

Comment (plttn) [reply azure-container-apps-arm/README.md]: Looks good from Solutions, deferring approval to AOP 😄

**Review by plttn — APPROVED** [review]

**Review by Hemal1P — APPROVED** [review]
Lgtm 🚀 
Will update the release notes link in description once ready.

**Pull request merged**